src/js/containers/account/awards/AccountAwardsContainer.jsx

```diff
@@ -302,13 +296,16 @@ export class AccountAwardsContainer extends React.Component {
         };
 
         const currentSortField = this.state.sort.field;
-
         // check if the current sort field is available in the table type
-        const availableFields = AccountTableSearchFields[tab]._mapping;
-        if (!availableFields[currentSortField]) {
+        const availableFields = this.state.columns[tab].data;
+        if (!{}.hasOwnProperty.call(availableFields, currentSortField)) {
```

Comment:
wut?

Reply:
dunno, just moved what we are using in advanced search, yeah, made me read it twice too, does not really make sense, we can go over it on Monday.

Reply:
This is really weird tho...